recipes/r-hash/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: r-hash
+  version: "2.2.6"
+
+source:
+  fn: hash_2.2.6.tar.gz
+  url: http://cran.r-project.org/src/contrib/hash_2.2.6.tar.gz
+  md5: d9f96c888b81b56a87aebbbe1f4d421a
+
+build:
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build: 
+    - r-base
+
+  run: 
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('hash')" # [not win]
```

Comment:
If you don't support Windows, you can remove the ` # [not win]` selector (which would fix the linting issue) and add `skip: true  # [win]` to the `build:` section.
Otherwise, you need an extra space before the `#` here and you also have to add a `build.bat` file.